docs/content/preview/explore/_index.md

```diff
@@ -226,6 +226,34 @@ ycqlsh>
   </div>
 </div>
 
+## Set up YB Simulation Base Demo
+
+YB Simulation Base Demo is a Java application that simulates workloads against YugabyteDB. Some Explore topics use the application to demonstrate features of YugabyteDB.
+
+Download the Simulation Base Demo JAR file (`yb-simu-base-app.jar`) using the following command:
+
+```sh
+wget https://github.com/yugabyte/yb-simulation-base-demo-app/releases/download/1.0/yb-simu-base-app.jar
+```
+
+To start the application against a running local cluster, use the following command:
+
+```sh
+java -jar ./yb-simu-base-app.jar
+```
+
+By default, the application connects to the local cluster at 127.0.0.1, adds tables, seeds some data, and starts a workload.
```

Comment:
```suggestion
By default, the application connects to the local cluster at 127.0.0.1 and from the UI, you can add tables, seed some data, and start a workload.
```

Reply:
This statement gives me a sense that everything is taken care of when we start the jar.